net/mwan3/files/usr/libexec/rpcd/mwan3

```diff
@@ -69,6 +69,26 @@ report_policies_v6() {
 	done
 }
 
+get_online_time() {
+	local time_n time_u iface
+	iface="$1"
+	time_u="$(cat "$MWAN3TRACK_STATUS_DIR/${iface}/ONLINE")"
+	[ -z "${time_u}" ] || [ "${time_u}" = "0" ] || {
+		time_n="$(get_uptime)"
+		echo $((time_n-time_u))
+	}
+}
+
+get_offline_time() {
+	local time_n time_d iface
```

Comment:
it it better to move these functions to `lib/mwan3/common.sh`

Reply:
I disagree. These are not used by anything else, and we should keep common.sh small to reduce memory footprint of other scripts.